content/rs/references/cli-utilities/crdb-cli/crdb/create.md

```diff
@@ -42,36 +43,70 @@ Before you create an Active-Active database, you must have:
+| oss-cluster                                                                           | true<br>**false**                               | Activates [OSS cluster API]({{< relref "/rs/administering/designing-production/networking/using-oss-cluster-api" >}})                                                                                                                                |
+| shard-key-regex \<regex_rule\>                                                       | string                                          | If clustering is enabled, this defines a regex rule (also known as a [hashing policy]({{< relref "/rs/concepts/high-availability/clustering#custom-hashing-policy" >}}) that determines which keys are located in each shard |
+| bigstore                                                                              | true <br> <br> **false**                        | If true, the database uses Reids on Flash to add Flash memory to the database                                                                                                                                                |
+| bigstore-ram-size \<size\>                                                           | size in bytes, kilobytes (KB) or gigabytes (GB) | Maximum RAM limit for the Redis on Flash database                                                                                                                                                                            |
+| --with-module name=\<module_name\>,version=\<module_version\>,args=\<module_args\> | strings                                         | Creates a database with a specific module                                                                                                                                                                                    |
+| eviction-policy                                                     | noeviction<br>allkeys-lru<br>allkeys-lfu<br>allkeys-random<br>volatile-lru<br>volatile-lfu<br>volatile-random<br>volatile-ttl | Sets [Eviction policy]({{< relref "/rs/databases/configure/eviction-policy" >}})                                                                                                          |
+| proxy-policy                                                                          | all-nodes<br>all-master-shards<br>single        | Sets Proxy policy |
+
 
 
 ### Returns
 
-Returns the task number of the task that is creating the database. If --no-wait is specified, the command exits. Otherwise, it will wait for the database to be created and then return the CRDB GUID.
+Returns the task ID of the task that is creating the database. If --no-wait is specified, the command exits. Otherwise, it will wait for the database to be created and then return the CRDB GUID.
```

Comment:
```suggestion
Returns the task ID of the task that is creating the database.

If `--no-wait` is specified, the command exits. Otherwise, it will wait for the database to be created and then return the CRDB GUID.
```